readthedocs_theme/templates/includes/footer.html

```diff
@@ -26,34 +26,35 @@ <h4 class="ui sub header">Stay updated</h4>
               <h4 class="ui sub header">Learn more</h4>
               <a class="item" href="https://docs.readthedocs.io">Documentation</a>
               <a class="item" href="https://docs.readthedocs.io/page/tutorial/index.html">Getting started guide</a>
-              <a class="item" href="https://docs.readthedocs.io/page/config-file/v2.html">Configure your project</a>
+              <a class="item" href="https://docs.readthedocs.io/page/config-file/index.html">Configure your project</a>
+              <a class="item" href="/comparisons/github-pages/">Comparison to GitHub Pages</a>
             {% endblock menu_learn %}
           </div>
         </div>
 
         <div class="column">
           <div class="ui vertical inverted text menu">
             {% block menu_services %}
-              <h4 class="ui sub header">Services</h4>
-              <a class="item" href="/pricing/">Pricing</a>
+              <h4 class="ui sub header">Content</h4>
               <a class="item" href="/about/">Features</a>
```

Comment:
We can probably rename this page too, it doesn't need to stay "/about/", and I'm not sure we have any links to it directly either.

Reply:
Yea, that's the same page, I've pushed a PR renaming it.

Reply:
Isn't this that PR? The link name is updated here, but it looks like the page is still `/about/`. I'm describing updating this link to `/features/` and moving `content/pages/about.html` to `content/pages/features.html`

Reply:
Updates with the page source file rename and page edits look good :+1: